perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -2487,6 +2487,11 @@ public void updateSponsorshipValidity(PerunSession sess, Member sponsoredMember,
 		perunBl.getAuditer().log(sess, new SponsorshipValidityUpdated(sponsoredMember, sponsor, newValidity));
 	}
 
+	@Override
+	public List<Sponsorship> getSponsorshipsExpiringInRange(PerunSession sess, LocalDate from, LocalDate to) {
+		return membersManagerImpl.getSponsorshipsExpiringInRange(sess, from, to);
```

Comment:
Nit: You could add null checks for from and to variables.

Reply:
Done.